### PR TITLE
Add github CI

### PR DESCRIPTION
**Pull request by DrSensor**
> blocked by: #2

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM gcr.io/distroless/static
+
+COPY target/release/darkpools_publicapi_ws_pairs /bin/
+
+CMD ["/bin/darkpools_publicapi_ws_pairs"]
```
Comment (Ujang360) [inline Dockerfile]: This is really interesting, I highly suggest we use alpine image.

Comment (DrSensor) [reply Dockerfile]: How about both 😏
- `publicapi_ws_pairs:distroless` -> for bare minimum image (no pkg manager)
- `publicapi_ws_pairs:alpine` -> for minimum image but user/developer still have flexibility to access shell and install package via `apk`

<sup>[my victim](https://github.com/nushell/nushell/blob/master/docs/docker.md)</sup>

But I will use alpine first for starter then gradually add another when there is no priority 😉

**Comment by DrSensor** [comment]
Btw, there is an alternative of [hub.docker.com](https://hub.docker.com) called [quay.io](https://quay.io) in case anybody interested.

[Example](https://quay.io/repository/drsensor/nu?tab=tags)

File: Dockerfile (hunk 1 above)
Comment (Ujang360) [reply Dockerfile]: Good point. But if we're going to use `distro-less`, these environment variables must be set as default:
https://github.com/darkpools/darkpools_publicapi_ws_pairs/blob/17e61162a7c4628a53b5d7b42a1de7372a533d08/run.sh#L24-L32

**Comment by Ujang360** [comment]
> Btw, there is an alternative of [hub.docker.com](https://hub.docker.com) called [quay.io](https://quay.io) in case anybody interested.
> 
> [Example](https://quay.io/repository/drsensor/nu?tab=tags)

Interesting, what about `Gihub Package Registry`?

**Comment by Ujang360** [comment]
> 
> Interesting, what about `Github Package Registry`?

Nevermind, we have that in this PR :joy:

**Pull request closed without merging**